### ISSUE TITLE
IDLE doc improvements

### ISSUE THREAD

**Issue by terryjreedy** [python/cpython#82006]
BPO | [37825](https://bugs.python.org/issue37825)
--- | :---
Nosy | @terryjreedy, @taleinat

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/terryjreedy'
closed_at = None
created_at = <Date 2019-08-11.16:54:19.592>
labels = ['expert-IDLE', '3.7', '3.8', '3.9']
title = 'IDLE doc improvements'
updated_at = <Date 2019-09-10.02:45:02.350>
user = 'https://github.com/terryjreedy'
```

bugs.python.org fields:
```python
activity = <Date 2019-09-10.02:45:02.350>
actor = 'terry.reedy'
assignee = 'terry.reedy'
closed = False
closed_date = None
closer = None
components = ['IDLE']
creation = <Date 2019-08-11.16:54:19.592>
creator = 'terry.reedy'
dependencies = []
files = []
hgrepos = []
issue_num = 37825
keywords = []
message_count = 5.0
messages = ['349399', '349427', '349435', '349436', '351565']
nosy_count = 2.0
nosy_names = ['terry.reedy', 'taleinat']
pr_nums = []
priority = 'normal'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue37825'
versions = ['Python 3.7', 'Python 3.8', 'Python 3.9']
```

</p></details>

**Comment by terryjreedy** [comment]
Points 4 and 5 are already done.  Upcoming PR addresses 1 and 2.

**Comment by terryjreedy** [comment]
Glitch in original patch fixed in #94366 and #94369 and in 3.10 backport of original.
